_data/projects/AsyncMonolith.yml

```
@@ -0,0 +1,12 @@
+name: AsyncMonolith
+desc: Facilitates simple asynchronous messaging in dotnet apps.
+site: https://github.com/Timmoth/AsyncMonolith
+tags:
+  - messaging
+  - scheduling
+  - jobs
+  - tasks
+  - events
+upforgrabs:
+  name: help wanted
+  link: https://github.com/timmoth/asyncmonolith/labels/help%20wanted
```

:art: casing for consistency

```suggestion
  link: https://github.com/Timmoth/AsyncMonolith/labels/help%20wanted
```